src/Compilers/CSharp/Portable/BoundTree/TupleBinaryOperatorInfo.cs

```diff
@@ -111,5 +111,37 @@ internal override TreeDumperNode DumpCore()
             }
 #endif
         }
+
+        /// <summary>
+        /// Represents an element-wise null/null comparison.
+        /// For instance, `(null, ...) == (null, ...)`.
+        /// </summary>
+        internal class NullNull : TupleBinaryOperatorInfo
+        {
+            internal readonly BinaryOperatorKind Kind;
+
+            internal NullNull(BinaryOperatorKind kind, TypeSymbol leftConvertedTypeOpt, TypeSymbol rightConvertedTypeOpt)
```

Comment:
>TypeSymbol leftConvertedTypeOpt, TypeSymbol rightConvertedTypeOpt [](start = 55, length = 65)

Are these always null? If so, consider removing these parameters and simply passing nulls to the base